src/services/expenses.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+let expenses = [];
+
+function getFiltered(data) {
+  const { userId, categories, from, to } = data;
+
+  if (userId) {
+    expenses = expenses.filter(expense => +userId === expense.userId);
```

Comment:
use `Number` instead `+`/
in all cases
```suggestion
    expenses = expenses.filter(expense => Number(userId) === expense.userId);
```